pod/perlexperiment.pod

```diff
@@ -162,6 +152,15 @@ Using this feature triggers warnings in the category C<experimental::try>.
 The ticket for this experiment is
 L<[perl #18760]|https://github.com/Perl/perl5/issues/18760>
 
+=item Use of C<@_> within subroutine signatures
+
+Introduced in Perl 5.36.0 as part of a reduction in the scope of experimental
+subroutine signatures.
+
+Using the default arguments array (C<@_>) within a subroutine that uses
+signatures will emit a warning in the category
+C<experimental::args_array_with_signatures>.
```

Comment:
I'd duplicate the information mentioned previously in case someone sees this and doesn't see the perldelta mention.

```
Using the default arguments array (C<@_>) within a subroutine that uses
signatures will emit a warning in the category
C<experimental::args_array_with_signatures>.

This includes C<@_>, C<$_[INDEX]>, or using C<shift>, C<unshift>, C<pop>,
or C<push> without a list of arguments.
```